tests/unittests/BackendTestUtils.cpp

```diff
@@ -57,6 +58,58 @@ void inferBatchedAddNet(Tensor *batch, Tensor *slice, Tensor *out,
   out->copyFrom(&result->getVariable()->getPayload());
 }
 
+/// A helper function to test quantized element-wise ops on the specific
+/// backend.
+void quantizedElementwiseOpsOnSpecificBackend(BackendKind backend, Tensor *lhs,
```

Comment:
I think this is not needed.

OperatorTest already covers quantized math for CPU and Interpreter. There is a parameter in the test which defines the backend. So the only thing you need to do is to enable test on OpenCL, e.g., TEST_P(Operator) instead of TEST_P(InterpAndCPU).

Reply:
moved it back to OperatorTest.cpp. I've changed the test to compare quantized op result with non-quantized op results. This saves me from previous problem of explicitly specifying backend.